hack/.vendor-helpers.sh

```diff
@@ -112,7 +112,7 @@ clean() {
 	local prune=( $(find vendor -depth -type d -not '(' "${findArgs[@]}" ')') )
 	unset IFS
 	for dir in "${prune[@]}"; do
-		find "$dir" -maxdepth 1 -not -type d -exec rm -v -f '{}' +
+		find "$dir" -maxdepth 1 -not -type d -not -name "LICENSE*" -exec rm -v -f '{}' +
```

Comment:
Yes I can switch to single, and sure we can add `COPYING*`, I am unaware of the significance of that file though